src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -1760,6 +1760,67 @@ .maxstack 8
             );
     }
 
+    [Fact]
+    public void Member_InstanceIndexer_Metadata()
```

Comment:
>Member_InstanceIndexer_Metadata [](http://example.com/codeflow?start=16&length=31)

It feels like this test isn't very useful right now. At least in the form that uses IL. I think we can compile an indexer declaration to metadata.